async function throwError() {
    return Promise.reject(Error('hello'));
}

